integrations/tensorflow/compiler/LegalizeTF.cpp

```diff
@@ -0,0 +1,124 @@
+// Copyright 2020 Google LLC
```

Comment:
general question: upstream isn't interested in this extension?

Reply:
On the TF side they assume they have access to the TF folders which are executed in a prior pass. We are in a special situation because we don't build the TF constant folders.